_posts/2017-04-27-Rust-1.17.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: post
+title: "Announcing Rust 1.17"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.17.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.17 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.17.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1170-2017-04-27
+
+### What's in 1.17.0 stable
+
+The story of Rust 1.17.0 is mostly one of small, quality of life improvements. For example,
+[the `'static` lifetime is now assumed in statics and consts](https://github.com/rust-lang/rust/pull/39265). When writing a const or static like this:
+
+```rust
+const NAME: &'static str = "Ferris";
+static NAME: &'static str = "Ferris";
+```
+
+Rust 1.17 will allow you to elide the `'static`, since that's the only lifetime that makes
+sense:
+
+```rust
+const NAME: &str = "Ferris";
+static NAME: &str = "Ferris";
+```
+
+In some situations, this can remove lots of boilerplate:
+
+```rust
+// old
+const NAMES: &'static [&'static str; 2] = &["Ferris", "Bors"];
+
+// new
+const NAMES: &[&str; 2] = &["Ferris", "Bors"];
+```
+
+Another similar improvement is "field init shorthand." Similar to ECMAScript 6,
+which calls this "Object Literal Property Value Shorthand", duplication can be
+removed when declaring structs, like this:
+
+```rust
+// definitions
+struct Point {
+    x: i32,
+    y: i32,
+}
+
+let x = 5;
+let y = 6;
+
+// old
+let p = Point {
+    x: x,
+    y: y,
+};
+
+// new
+let p = Point {
+    x,
+    y,
+};
+```
+
+That is, the `x, y` form will assume that its values are set to a varaible
+with the same name in its scope.
+
+In a similar way, it's common for new Rustaceans to try to use `+` to add two
```

Comment:
yeah, "similar" here was the overall topic of the release: small ergonomic changes. I'll re-word.